web/src/builder.rs

```diff
@@ -0,0 +1,631 @@
+        };
+    }
+
+    #[wasm_bindgen(js_name = "setScale")]
+    pub fn set_scale(&mut self, value: &str) {
+        self.scale = match value {
+            "exactfit" => StageScaleMode::ExactFit,
+            "noborder" => StageScaleMode::NoBorder,
+            "noscale" => StageScaleMode::NoScale,
+            "showall" => StageScaleMode::ShowAll,
+            _ => return,
+        };
+    }
+
+    #[wasm_bindgen(js_name = "setForceScale")]
+    pub fn set_force_scale(&mut self, value: bool) {
+        self.force_scale = value;
+    }
+
+    #[wasm_bindgen(js_name = "setFrameRate")]
+    pub fn set_frame_rate(&mut self, value: Option<f64>) {
+        self.frame_rate = value;
+    }
+
+    #[wasm_bindgen(js_name = "setWmode")]
+    pub fn set_wmode(&mut self, value: Option<String>) {
+        self.wmode = value;
+    }
+
+    #[wasm_bindgen(js_name = "setLogLevel")]
+    pub fn set_log_level(&mut self, value: &str) {
+        if let Ok(level) = tracing::Level::from_str(value) {
+            self.log_level = level;
+        }
+    }
+
+    #[wasm_bindgen(js_name = "setMaxExecutionDuration")]
+    pub fn set_max_execution_duration(&mut self, value: f64) {
+        self.max_execution_duration = Duration::from_secs_f64(value);
+    }
+
+    #[wasm_bindgen(js_name = "setPlayerVersion")]
+    pub fn set_player_version(&mut self, value: Option<u8>) {
+        self.player_version = value;
+    }
+
+    #[wasm_bindgen(js_name = "setPreferredRenderer")]
+    pub fn set_preferred_renderer(&mut self, value: Option<String>) {
+        self.preferred_renderer = value;
+    }
+
+    #[wasm_bindgen(js_name = "setOpenUrlMode")]
+    pub fn set_open_url_mode(&mut self, value: &str) {
+        self.open_url_mode = match value {
+            "allow" => OpenURLMode::Allow,
+            "confirm" => OpenURLMode::Confirm,
+            "deny" => OpenURLMode::Deny,
+            _ => return,
+        };
+    }
+
+    #[wasm_bindgen(js_name = "setAllowNetworking")]
+    pub fn set_allow_networking(&mut self, value: &str) {
+        self.allow_networking = match value {
+            "all" => NetworkingAccessMode::All,
+            "internal" => NetworkingAccessMode::Internal,
+            "none" => NetworkingAccessMode::None,
+            _ => return,
+        };
+    }
+
+    #[wasm_bindgen(js_name = "addSocketProxy")]
+    pub fn add_socket_proxy(&mut self, host: String, port: u16, proxy_url: String) {
+        self.socket_proxy.push(SocketProxy {
+            host,
+            port,
+            proxy_url,
+        })
+    }
+
+    #[wasm_bindgen(js_name = "setCredentialAllowList")]
+    pub fn set_credential_allow_list(&mut self, value: Vec<String>) {
+        self.credential_allow_list = value;
+    }
+
+    #[wasm_bindgen(js_name = "setPlayerRuntime")]
+    pub fn set_player_runtime(&mut self, value: &str) {
+        self.player_runtime = match value {
+            "air" => PlayerRuntime::AIR,
+            "flash_player" => PlayerRuntime::FlashPlayer,
```

Comment:
Uhh, isn't this a breaking change? (It was `flashPlayer` before)

Reply:
You're right, thank you!